contrib/epee/src/net_ssl.cpp

```diff
@@ -43,6 +43,10 @@
 // openssl req -new -key /tmp/KEY -out /tmp/REQ
 // openssl x509 -req -days 999999 -sha256 -in /tmp/REQ -signkey /tmp/KEY -out /tmp/CERT
 
+#ifdef _WIN32
+static void add_windows_root_certs(boost::asio::ssl::context &ctx);
```

Comment:
Real bikeshedding - can the whole definition be put here? Not sure why it was split.

Reply:
It did not compile if the whole body was first. Something #defines something I think but since it was a report from someone else I did not investigate what exactly.